docs/content/manual/manual.yml

```diff
@@ -1368,13 +1368,21 @@ sections:
           `sort_by(foo)` compares two elements by comparing the result of
           `foo` on each element.
 
+          Also note that `sort_by(.k1, .k2, .k3, ...)` behaves as you would
+          expect: it sorts first by comparing elements by key `k1`, then
+          comparing elements by key `k2` to "break ties" left by `k1`,
+          then comparing elements by key `k3`, and so on.
+
         examples:
           - program: 'sort'
             input: '[8,3,null,6]'
             output: ['[null,3,6,8]']
           - program: 'sort_by(.foo)'
             input: '[{"foo":4, "bar":10}, {"foo":3, "bar":100}, {"foo":2, "bar":1}]'
             output: ['[{"foo":2, "bar":1}, {"foo":3, "bar":100}, {"foo":4, "bar":10}]']
+          - program: 'sort_by(.foo, .bar)'
+            input: '[{"foo":4, "bar":10}, {"foo":3, "bar":100}, {"foo":2, "bar":1}, {"foo":3, "bar":200}]'
+            output: ['[{"foo":2, "bar":1}, {"foo":3, "bar":100}, {"foo":3, "bar":200}, {"foo":4, "bar":10}]']
```

Comment:
The input value can be improved by flipping the foo:3 elements to make it clear the program sorts by bar (program `sort_by(.foo)` yields the same output currently).

Reply:
BTW, it would also be great to have another complicated example where the keys have transformations. For example:
```console
$ echo '[{"foo":"20", "bar":10}, {"foo":"3", "bar":100}, {"foo":"2", "bar":1}, {"foo":"3", "bar":200}]' | jq '. | sort_by([.foo | tonumber], .bar)'
[
  {
    "foo": "2",
    "bar": 1
  },
  {
    "foo": "3",
    "bar": 100
  },
  {
    "foo": "3",
    "bar": 200
  },
  {
    "foo": "20",
    "bar": 10
  }
]
```